packages/terra-form-select/src/single/Menu.jsx

```diff
@@ -161,16 +161,9 @@ class Menu extends React.Component {
     this.clearSearch();
     this.clearScrollTimeout();
     if (this.state.closedViaKeyEvent) {
-      this.props.select.focus();
-      if (SharedUtil.isSafari()) {
-        /**
-         * Shifting focus back to the select specifically
-         * when VoiceOver is on will sometimes trigger VO to shift focus
-         * randomly to the root of document or the skip to main link
-         * instead of the select and then break VoiceOver usage when navigating the
-         * select options on subsequent opening of select
-         * Refocusing on select seems to seems to mitigate this VO bug.
-         */
+      if (!SharedUtil.isSafari()) {
```

Comment:
Just for confirmation was this code change made to improve readability or to address any specific issue. Because the above code changes is same as previous code where timeOut is added only for safari and this.props.select.focus(); is for other browsers. 

```jsx
if (this.state.closedViaKeyEvent) {
      this.props.select.focus();
      if (SharedUtil.isSafari()) {
        /**
         * Shifting focus back to the select specifically
         * when VoiceOver is on will sometimes trigger VO to shift focus
         * randomly to the root of document or the skip to main link
         * instead of the select and then break VoiceOver usage when navigating the
         * select options on subsequent opening of select
         * Refocusing on select seems to seems to mitigate this VO bug.
         */
        setTimeout(() => {
          this.props.select.focus();
        }, 300);
      }
    }
```

Reply:
Previously `this.props.select.focus();` was done for all browsers (including safari) and then after some time delay it was getting done for safari again. So basically for safari, it was setting up focus twice because of that the screen reader was somtimes not annoucing that the list item was selected. It was just skipping that.